README.md

```diff
@@ -0,0 +1,99 @@
+# terraform-aws-emr-cluster
+
+A Terraform module to create an Amazon Web Services (AWS) Elastic MapReduce (EMR) cluster.
+
+## Usage
+
+```hcl
+data "template_file" "emr_configurations" {
+  template = "${file("configurations/default.json")}"
+}
+
+module "emr" {
+  source = "github.com/azavea/terraform-aws-emr-cluster?ref=0.1.0"
+
+  name          = "DatarpocCluster"
+  vpc_id        = "vpc-20f74844"
```

Comment:
Not sure if this is an actual VPC id but I figured I'd draw attention to it anyway.

Reply:
I just mashed the keyboard.